packages/types/src/coreExtensions/componentEditor.ts

```diff
@@ -1,5 +1,40 @@
 type Patcher<T> = (elements: React.ReactNode[], props: T) => React.ReactNode[];
 
+//#region Chat Buttons
+// eslint-disable-next-line prettier/prettier
```

Comment:
I would prefer to not ignore this - even if Prettier is gonna be wonky about it, I don't like scattering ignores around the codebase.

Reply:
Fixed this.